acceptance/pe/tests/basic.rb

```diff
@@ -0,0 +1,20 @@
+require 'scooter'
+
+test_name "basic validation of puppetdb-cli subcommands" do
+  puppet_query_on(client, "--help")
+  puppet_db_on(client, "--help")
+
+  step 'setup user and get token with puppet-access' do
+    console_dispatcher = Scooter::HttpDispatchers::ConsoleDispatcher.new(master)
+    administrator_role = console_dispatcher.get_role_by_name('Administrators')
+    user = console_dispatcher.generate_local_user
+    console_dispatcher.add_user_to_role(user, administrator_role)
+    login_with_puppet_access_on(client, user)
+  end
+
+  puppet_query_on(client, "'nodes{}'")
+  puppet_db_on(client, "status")
+  dir = create_tmpdir_on(client)
+  puppet_query_on(client, "export #{dir}/pdb_archive.tgz")
```

Comment:
I am not sure what the correct behavior is for this line. If I run it as is, `puppet-query` returns an `invalid arguments` error which is what I would expect.
```
   /opt/puppetlabs/client-tools/bin/puppet-query export /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/pdb-cli-basic.XXXXXX.XehX2gwl/pdb_archive.tgz
  Last 10 lines of output were:
        Invalid arguments.

        Usage:
          puppet-query [options] (--version | --help)
          puppet-query [options] <query>>
      Test line: pe/tests/basic.rb:18:in `block (2 levels) in run_test'
```

If I assume that the `export` command is a query and single quote it, I get a PQL parse error:
```
/opt/puppetlabs/client-tools/bin/puppet-query 'export /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/pdb-cli-basic.XXXXXX.xHFV4NvP/pdb_archive.tgz'
    Error response from server: PQL parse error at line 1, column 1:

    export /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/pdb-cli-basic.XXXXXX.xHFV4NvP/pdb_archive.tgz
    ^

    Expected one of:

    events
    edges
    inventory
    fact_contents
    fact_paths
    factsets
    environments
    nodes
    resources
    catalogs
    reports
    facts
    #"\s+"
```